integration/testnet/network.go

```diff
@@ -609,8 +615,7 @@ func PrepareFlowNetwork(t *testing.T, networkConf NetworkConfig, chainID flow.Ch
 	})
 
 	// create a temporary directory to store all bootstrapping files
-	bootstrapDir, err := integrationBootstrapDir()
-	require.Nil(t, err)
+	bootstrapDir := tempDir(t)
```

Comment:
can https://pkg.go.dev/testing#T.TempDir be used instead?

Reply:
`TempDir` created directories which couldn't be written to later, causing permissions issues failing the tests - hence the consolidation on the method here: https://github.com/onflow/flow-go/pull/3846

Reply:
Ah! Interesting, good to know, thanks.